src/pages/InputNewTea.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import InputForm from '../components/InputForm'
+
+export default function InputNewTea({ setTea }) {
+  return (
+    <>
```

Comment:
Da du hier momentan nur eine Komponente aufrufst die in sich abgeschlossen ist, brauchst du die Fragment-tags hier nicht. Aber siehe mein Kommentar unten, wenn du das machst bräuchtest du sie wieder.